vocode/streaming/models/agent.py

```diff
@@ -115,6 +115,7 @@ class ChatGPTAgentConfig(AgentConfig, type=AgentType.CHAT_GPT.value):  # type: i
     openai_api_key: Optional[str] = None
     prompt_preamble: str
     model_name: str = CHAT_GPT_AGENT_DEFAULT_MODEL_NAME
+    base_url: Optional[str] = None
```

Comment:
let's either name this `base_url_override` or default it to `"https://api.openai.com/v1"` — i'd prefer the former since it changes the code less

Reply:
Hi @ajar98 nice to meet you! Thanks for looking over this.

I just pushed some changes to go to the override option. Let me know if anything else needs to get changed!